core/dbt/contracts/graph/model_config.py

```diff
@@ -433,6 +433,7 @@ class NodeConfig(NodeAndTestConfig):
     # Note: if any new fields are added with MergeBehavior, also update the
     # 'mergebehavior' dictionary
     materialized: str = "view"
+    incremental_strategy: Optional[str] = None
```

Comment:
@gshank  @nathaniel-may does this now mean that the value of `incremental_strategy` is now in the manifest.json when it wasn't before? or does this add it to the python context such that it is accessible as an attribute of `model` as in `model.incremental_strategy`?

Reply:
It means that it should be in the manifest.json. It's accessible like other config keys, but the behavior is a bit different for builtin attributes than for adhoc attributes, in that setting defaults doesn't work the same way.